targettests/execution/quantum_struct_exec.cpp

```diff
@@ -0,0 +1,37 @@
+/*******************************************************************************
+ * Copyright (c) 2022 - 2024 NVIDIA Corporation & Affiliates.                  *
+ * All rights reserved.                                                        *
+ *                                                                             *
+ * This source code and the accompanying materials are made available under    *
+ * the terms of the Apache License 2.0 which accompanies this distribution.    *
+ ******************************************************************************/
+
+// RUN: nvq++ %cpp_std --enable-mlir %s -o %t && %t | FileCheck %s
+
+#include "cudaq.h"
+
+struct test {
+  int i;
+  double d;
+  cudaq::qview<> q;
+};
+
+__qpu__ void hello(cudaq::qubit &q) { h(q); }
+
+__qpu__ void kernel(test t) {
```

Comment:
Passing a copy of an object that contains a `qview<>` here ... seems like it ought to be passing this by reference.

Reply:
`qview<>` is meant to be passed by value (`std::span` like)